### PR TITLE
Added the list of GPS part numbers to the GPS interface

### PR DESCRIPTION
**Pull request by Fberbari**
Added Gps part number list.

### CODE REVIEW

**Review by heyitslu — APPROVED** [review]

File: Autopilot/Inc/OutputMixing.hpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+/***********************************************************************************************************************
+ * Definitions
+ **********************************************************************************************************************/
+
+typedef struct
+{
+	int errorCode;
+
+}OutputMixing_eror_t;
```
Comment (heyitslu) [inline Autopilot/Inc/OutputMixing.hpp]: typo oop lol

File: Autopilot/Inc/SendToActuators.hpp [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+/***********************************************************************************************************************
+ * Definitions
+ **********************************************************************************************************************/
+
+typedef struct
+{
+	int errorCode;
+
+}SendToActuators_eror_t;
```
Comment (heyitslu) [inline Autopilot/Inc/SendToActuators.hpp]: same typo - error needs 2 r's

File: Autopilot/Inc/OutputMixing.hpp (hunk 1 above)
Comment (Fberbari) [reply Autopilot/Inc/OutputMixing.hpp]: lol good catch

File: Autopilot/Inc/SendToActuators.hpp [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+*/
+void SendToActuators_Init(void);
+
+/**
+* Communicates the desired actuator commands encoded in the channelOut array over PWM to
+* the appropriate actuator channels.
+* @param[in]		channelOut 		the array (size of 4 floats) of percentages each channel should be set to.
+* @return							the error struct, containing all info about any errors that may have occured.
+*/
+SendToActuators_eror_t SendToActuators_Execute(float *channelOut);
```
Comment (heyitslu) [inline Autopilot/Inc/SendToActuators.hpp]: typo pt 2

File: Autopilot/Inc/OutputMixing.hpp [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+ **********************************************************************************************************************/
+
+/**
+* Converts the desired roll, pitch, yaw, and thrust percentages into an array of percentages
+* corresponding directly to what percentage of full the actuator attached to each channel should be set to.
+* @param[in]		PidOutput 		pointer to the struct containing the desired attitude percentages.
+* @param[out]		channelOut 		the array (size of 4 floats) of percentages each channel should be set to.
+* @return							the error struct, containing all info about any errors that may have occured.
+*/
+OutputMixing_eror_t OutputMixing_Execute(PID_Output_t *PidOutput, float *channelOut);
```
Comment (heyitslu) [inline Autopilot/Inc/OutputMixing.hpp]: typo pt 2

**Review by heyitslu — APPROVED** [review]
Couple of minor typos but everything else looks good

**Review by avlepage — COMMENTED** [review]
What lu said about the typos, otherwise looks good!

File: Autopilot/Inc/SendToActuators.hpp (hunk 3 above)
Comment (Fberbari) [reply Autopilot/Inc/SendToActuators.hpp]: Jesus christ, my bad lmao.

**Pull request closed without merging**